conn-fix/README.md

```diff
@@ -0,0 +1,359 @@
+# FIX Connect
+
+This "Connect" component extension supports connection to the target system via [FIX protocol](https://www.fixtrading.org/what-is-fix/).
+
+## Available service types
+
+This "Connect" component provides users with two types of the services:
+
+### FIX Client
+
+The "Connect" component acts like a client that connects to the system.
+The configuration type that needs to be used is - **FIX_Client**.
+
+#### Configurable parameters
+
+The FIX Client has the following parameters to configure:
+```yaml
+# It is a required parameter. It's the FIX version that this session should use.
+# Possible values: "FIX.4.0", "FIX.4.1", "FIX.4.2", "FIX.4.3", "FIX.4.4", "FIXT.1.1"
+beginString: FIXT.1.1
+
+# It is a required parameter. Your ID that is associated with this FIX session.
+senderCompID: sender
+
+# It is a required parameter. Counter Parties ID that is associated with this FIX session.
```

Comment:
It is a required parameter. Counter Parties ID that **are** associated with this FIX session.